src/UI/button/Button.jsx

```diff
@@ -0,0 +1,10 @@
+import styles from './Button.scss';
+
+function Button(props) {
+  return (
+    <button {...props} className={styles.Btn}>
```

Comment:
spreading the props should usually be done last unless you specifically want something not to get overwritten